inline_scan.sh

```diff
@@ -21,7 +20,7 @@ FAILED_IMAGES=()
 VALIDATED_OPTIONS=""
 # Vuln scan option variable defaults
 DOCKERFILE="./Dockerfile"
-POLICY_BUNDLE="./policy_bundle.json"
+# shellcheck disable=SC2034
 TIMEOUT=300
```

Comment:
I am not sure if TIMEOUT is used at all! As it is an environment variable, it might be used by anchore engine, but I don't see any traces of "TIMEOUT" in the anchore-engine code, and it is not used in the script. I assume it could be safely removed.

Reply:
I leave it because it's passed over here:

https://github.com/sysdiglabs/secure-inline-scan/blob/add16705b03997ca6cb208dd8de8c327ed6a9cca/inline_scan.sh#L288

If you don't mind I'd rather fix this and your next comment in a different PR.